### ISSUE TITLE
If the interface can be more concise it'll be more perfect

### ISSUE THREAD

**Issue by playgithub** [Microsoft/CNTK#1632]
For example
```
from cntk import Trainer, learning_rate_schedule, UnitType
from cntk.learner import sgd
```
1. **trainer** and **learner**, two concepts here, seems that one is enough
2. **learning_rate_schedule(learning_rate, unit, epoch_size)**, it's not intuitive for the concept of "learning rate  schedule", why not decouple "learning_rate", "unit", and "epoch_size", so no need for  "learning rate  schedule"

**Comment by cha-zhang** [comment]
In Beta 15 we have updated the namespace. Currently we keep things as is for backward compatibility. Will introduce the breaking change in RC.
